### PR TITLE
made changes per flake8, updated tom notebook, fixed functions in tom…

### PR DESCRIPTION
**Pull request by m-kovalsky**
… model

### CODE REVIEW

File: src/sempy_labs/_ai.py [hunk 1]
```diff
@@ -367,7 +384,7 @@ def generate_aggs(
                 print(
                     f"{icons.green_dot} '{aggTableName}'[{fromColumn}] -> '{toTable}'[{toColumn}] relationship has been added."
                 )
-            except:
+            except Exception:
```
Comment (eisber) [inline src/sempy_labs/_ai.py:387]: do you want to swallow the exception here? at least I would surface the exception message as part of the print.

File: src/sempy_labs/migration/_create_pqt_file.py [hunk 2]
```diff
@@ -188,16 +194,16 @@ def obj_to_dict(obj):
...
-            types,
-            "{%s}Default" % ns,
-            {"Extension": "json", "ContentType": "application/json"},
-        )
-        default2 = ET.SubElement(
-            types,
-            "{%s}Default" % ns,
-            {"Extension": "pq", "ContentType": "application/x-ms-m"},
-        )
+        # default1 = ET.SubElement(
```
Comment (eisber) [inline src/sempy_labs/migration/_create_pqt_file.py:197]: do you want to disable this?

File: src/sempy_labs/migration/_migrate_tables_columns_to_semantic_model.py [hunk 3]
```diff
@@ -111,7 +111,7 @@ def migrate_tables_columns_to_semantic_model(
 
                         try:
                             tom.model.Tables[tName]
-                        except:
+                        except Exception:
```
Comment (eisber) [inline src/sempy_labs/migration/_migrate_tables_columns_to_semantic_model.py:114]: is an exception thrown if the table does not exist? can you check if the table exists first, then catching the exception. again this is masking all other errors.

**Review by eisber — CHANGES_REQUESTED** [review]

File: src/sempy_labs/migration/_create_pqt_file.py (hunk 2 above)
Comment (m-kovalsky) [reply src/sempy_labs/migration/_create_pqt_file.py:197]: it turns out these parameters were not used in the function.

File: src/sempy_labs/_ai.py (hunk 1 above)
Comment (m-kovalsky) [reply src/sempy_labs/_ai.py:387]: I updated the exception message to carry forward.

File: src/sempy_labs/migration/_migrate_tables_columns_to_semantic_model.py (hunk 3 above)
Comment (m-kovalsky) [reply src/sempy_labs/migration/_migrate_tables_columns_to_semantic_model.py:114]: updated this so it doesn't need a try/except clause

**Review by eisber — APPROVED** [review]

**Pull request merged**